replication/parser.go

```diff
@@ -23,14 +30,14 @@ type BinlogParser struct {
 	// used to start/stop processing
 	stopProcessing uint32
 
-	useDecimal bool
+	useDecimal     bool
+	verifyChecksum bool
 }
 
 func NewBinlogParser() *BinlogParser {
 	p := new(BinlogParser)
 
 	p.tables = make(map[uint64]*TableMapEvent)
-	// p.stop = make(uint32)
```

Comment:
why remove it?

Reply:
It is not used anywhere. If you want to keep it, I will put it back.

Reply:
I am very suprised that there is no stop field now but how can it pass compilation before....

Reply:
It's commented out before, and now I delete it completely.